lib/components/molecules/ArticleThumbnail/ArticleThumbnail.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+/**
+ *
+ * ArticleThumbnail
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './ArticleThumbnail.style';
+import type { Props } from './types';
+import Image from '../../atoms/Image/Image';
+import Anchor from '../../atoms/Anchor';
+
+const getAnchoredChild = (anchorProps, children) => <Anchor {...anchorProps}>{children}</Anchor>;
+
+const ArticleThumbnail = ({ image, readMoreContent, children, anchorProps }: Props): Node => {
+  let thumbImage = <Image {...image} />;
+
+  if (image.anchorProps) {
```

Comment:
Why there would be anchorProps inside image? 

@bhardwajamit0890 you just now to fix the data structure rest of the things looks good. Why this is important is: we need to keep the API consistent.